conftest.py

```diff
@@ -108,14 +118,6 @@ def run_and_wait(
         assert rv == 0 or rv["StatusCode"] == 0
         return logs
 
-    def get_host_port(self, container_port: str) -> str:
-        """Returns the host port associated with the tracked container's port."""
-        assert isinstance(self.container, Container)
-        self.container.reload()
```

Comment:
This doesn't actually work well when running many tests in parallel :=(
I don't really want to dig in the reason, but I think I've found an implementation, which works more reliable.